x-pack/metricbeat/module/cockroachdb/_meta/kibana/7/dashboard/Metricbeat-cockroachdb-overview.json

```diff
@@ -731,46 +731,11 @@
                 "metrics": [
                   {
                     "agg_with": "avg",
-                    "field": "prometheus.metrics.raft_process_logcommit_latency_count",
+                    "field": "prometheus.raft_process_logcommit_latency.histogram",
```

Comment:
Current dashboard is using sum and count to calculate the average of this value. I think it can make sense now to calculate percentiles, but I haven't managed to use histograms in TSVB yet. @exekias do you know if they are supported?

Reply:
It works with other visualizations.

Reply:
Yes, currently only Visualize supports this type

Reply:
I have replaced the graphs that were using sum and count to calculate averages and they are using 99th percentile now (as the CockroachDB admin UI does). It is quite ok now but the timings are in nanoseconds and I haven't found a way to format them.